src/config/configurator.ts

```diff
@@ -34,6 +34,17 @@ function readConfigFile(path: string): object {
     throw TypeError("Config file must be in YAML or JSON format");
   }
 }
+function setRemovalPolicy(value: any, node: cdk.CfnResource): void {
```

Comment:
Isn't the type of `value` a string?

Reply:
You're right this would always be a string, `cdk.RemovalPolicy.DESTROY` just translates to a string I believe